lib/semantics/type.h

```diff
@@ -280,7 +280,10 @@ class DeclTypeSpec {
   bool IsNumeric(TypeCategory) const;
   const NumericTypeSpec &numericTypeSpec() const;
   const LogicalTypeSpec &logicalTypeSpec() const;
-  const CharacterTypeSpec &characterTypeSpec() const;
+  const CharacterTypeSpec &characterTypeSpec() const {
```

Comment:
Why make this function different from the others in that it is in line? (I don't see a reason for any of them to be.)

Reply:
It makes the implementation of the function available to code in the evaluation library without introducing a cycle of dependency between the two shared binary libraries.